compare_platforms/compare_platforms_overflow.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * This example demonstrates a simple overflow of a pointer that has had
+ * 0x1001 bytes allocated using malloc(), and compares the different points
+ * at which the program will crash in several different architecture/OS environments:
+
+ * - x86_64 GNU/Linux
+ * - x86_64 Apple
+ * - arm64 Apple
+ * - riscv64-purecap
+ * - morello-purecap
+ * - morello-hybrid
```

Comment:
Are there individual points for each of these 6 architectures where the crash happens?

Reply:
There were, but now there are individual points for the 3 CHERI architectures only, as the timing of the crash is unpredictable in non-CHERI environments. See 6df379124a0c3a49f3d8a0b2a87fdea22697e50b.